logic/src/commonMain/kotlin/com/wire/kalium/logic/data/user/UserRepository.kt

```diff
@@ -291,6 +295,12 @@ internal class UserDataSource internal constructor(
             }
             .map(memberMapper::fromMapOfClientsToRecipients)
 
+    override suspend fun updateUserFromEvent(event: Event.User.Update): Either<CoreFailure, Unit> = wrapStorageRequest {
+        val userId = qualifiedIdMapper.fromStringToQualifiedID(event.userId!!)
```

Comment:
We shouldn't force unwrap the userId, but is the `userId` really optional on the user.update event?